desec/resource_rrset.go

```diff
@@ -44,7 +44,7 @@ func resourceRRSet() *schema.Resource {
 				Type:         schema.TypeString,
 				Required:     true,
 				ForceNew:     true,
-				ValidateFunc: validation.StringInSlice([]string{"A", "AAAA", "CAA", "CNAME", "TXT", "SRV", "LOC", "MX", "NS", "SPF", "CERT", "DNSKEY", "DS", "NAPTR", "SMIMEA", "SSHFP", "TLSA", "URI", "PTR"}, false),
+				ValidateFunc: validation.StringInSlice([]string{"A", "AAAA", "AFSDB", "APL", "CAA", "CERT", "CNAME", "DHCID", "DNAME", "DNSKEY", "DLV", "DS", "EUI48", "HINFO", "HTTPS", "KX", "L32", "L64", "LOC", "LP", "MX", "NAPTR", "NID", "NS", "OPENPGPKEY", "PTR", "RP", "SMIMEA", "SPF", "SRV", "SSHFP", "SVCB", "TLSA", "TXT", "URI"}, false),
```

Comment:
I vote for just removing this altogether since when it's out of date it's a blocker compared to things like TTL:
```suggestion
```

Reply:
yeah I tend to agree. the api will fail soon enough if an incorrect value is used here